src/components/OracleEventsQueryPanel.vue

```diff
@@ -4,21 +4,46 @@
       Query Details
     </div>
     <dl>
-      <dt>Query ID:</dt>
+      <dt class="oracle-events-query-panel__label">
+        Query ID:
+        <hint-tooltip>
+          {{ oraclesHints.queryId }}
+        </hint-tooltip>
+      </dt>
       <dd>{{ event.queryId }}</dd>
 
-      <dt>Query Fee:</dt>
+      <dt class="oracle-events-query-panel__label">
+        Query Fee:
+        <hint-tooltip>
+          {{ oraclesHints.queryFee }}
+        </hint-tooltip>
+      </dt>
       <dd>{{ formatAePrice(event.queryFee) }}</dd>
 
-      <dt>Query:</dt>
+      <dt class="oracle-events-query-panel__label">
+        Query:
+        <hint-tooltip>
+          {{ oraclesHints.queryContent }}
+        </hint-tooltip>
+      </dt>
       <dd class="oracle-events-query-panel__description">
         {{ event.query }}
       </dd>
 
-      <dt>Response TTL:</dt>
+      <dt class="oracle-events-query-panel__label">
+        Response TTL:
+        <hint-tooltip>
+          {{ oraclesHints.responseTtl }}
+        </hint-tooltip>
+      </dt>
       <dd>{{ event.responseTtl }}</dd>
 
-      <dt>Response:</dt>
+      <dt class="oracle-events-query-panel__label">
```

Comment:
Here I suggest to reuse the given name `term`, not `label`
https://www.w3schools.com/tags/tag_dt.asp

```suggestion
      <dt class="oracle-events-query-panel__term">
```